widgets/blueprints/src/BlueprintsCatalog.tsx

```diff
@@ -135,54 +135,54 @@ export default function BlueprintsCatalog({
                             </Grid.Row>
                         )}
                     </Grid>
-
-                    <Grid.Column width="16">
-                        <div style={{ height: '80px' }} />
-                    </Grid.Column>
-                </DataSegment.Item>
-
-                {BlueprintActions.isCompleted(item) && (
-                    <div className="actionButtons">
-                        <Button
-                            icon="trash"
-                            content="Delete"
-                            basic
-                            labelPosition="left"
-                            onClick={event => {
-                                event.stopPropagation();
-                                onDeleteBlueprint(item);
-                            }}
-                        />
-                        {BlueprintActions.isUploaded(item) && (
-                            <>
-                                <Button
-                                    icon="rocket"
-                                    content="Deploy"
-                                    labelPosition="left"
-                                    onClick={event => {
-                                        event.stopPropagation();
-                                        onCreateDeployment(item);
-                                    }}
-                                />
-
-                                {!manager.isCommunityEdition() && widget.configuration.showComposerOptions && (
+                    {BlueprintActions.isCompleted(item) && (
+                        <Grid style={{ marginTop: 'auto', paddingTop: '1rem' }}>
+                            <Grid.Row className="centered">
```

Comment:
Can we use here:
```suggestion
                            <Grid.Row centered>
```
? If so, then it's a bit cleaner solution IMHO.

Reply:
@qooban you're 100% correct, good catch, I'm gonna change it.